editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx

```diff
@@ -211,6 +211,7 @@ export default function Component(props: Props) {
                 osProxyEndpoint={`${process.env.REACT_APP_API_URL}/proxy/ordnance-survey`}
                 osCopyright={`Basemap subject to Crown copyright and database rights ${new Date().getFullYear()} OS (0)100024857`}
                 drawGeojsonDataCopyright={`<a href="https://www.planning.data.gov.uk/dataset/title-boundary" target="_blank" style="color:#0010A4;">Title boundary</a> subject to Crown copyright and database rights ${new Date().getFullYear()} OS (0)100026316`}
+                collapseAttributions={self.innerWidth < 500 ? true : undefined}
```

Comment:
```suggestion
                collapseAttributions={self.innerWidth < 500}
```

Would this work?

Reply:
It's a weird Lit "boolean" thing where the prop explitly needs to evaluate to `true` or `undefined` - `false` will be read as truthy (_existance_ of prop) by Lit !

Reply:
Aaaah that makes sense - we've hit this before. Thanks for the explanation 👍